### ISSUE TITLE
Piper maintainership status

### ISSUE THREAD

**Issue by whot** [libratbag/piper#444]
This issue is a note about Piper's maintainership status.

Right now, @whot is the official maintainer but he's seriously out of time to work on piper. Which means all but the simplest issues will go largely unanswered. Pull requests will be merged depending on difficulty - new SVGs and translations are easy, debugging complex and device-specific issues are neigh impossible.

This is unlikely to change unless we get more maintainers into the fold.

You can help out by reviewing patches/pull requests and helping users to debug the issues you're seeing.

You can help out by expanding the wiki to explain **how** to debug issues.

Please do so, otherwise Piper will just sit there largely stale for the foreseeable future.

**Comment by FFY00** [comment]
I tried to hack into piper a bit but I wasn't very successful. I will have to mess around more to figure out the internal workings but like you my time is also limited and split across several projects.

**Comment by stephanlachnit** [comment]
I think I can help out in my spare time. Maybe I will spend more time on libratbag/piper once my exams are over (February), but since the time effort for studying is somewhat inconsistent I can't promise that.

**Comment by LiamDawe** [comment]
Almost a year since this was posted, just wanted to check in to see what the current status is? Are new people still needed?

**Comment by FFY00** [comment]
Help is always welcome :blush:. There hasn't been any update since @stephanlachnit joined. I think we can currently manage maintaining the current software but adding new features or improvements might be slow.

**Comment by stephanlachnit** [comment]
> Almost a year since this was posted, just wanted to check in to see what the current status is? Are new people still needed?

I guess what we need the most is reviewers, probably more on the libratbag side than here.

**Comment by JoeJoeTV** [comment]
Has there been any development in regards to reviewers? Just wanted to kindly ask, because I really like this project but I can't help ecxept report bugs any maybe test some things.

**Comment by FFY00** [comment]
Hi, is there any particular work you are interested in? I could take some time to have a look if it's not too big or if it is high impact. It's a bit hard to manage life, OSS responsibilities, and OSS work I am exiting about.

**Comment by le-grand-mannitout** [comment]
Hello, I wanted to pr (#622) but I am struggling making a .svg, especially with colors and filling spaces (using gimp).
Can you please  tell me which software/method are you using to make each mouse .svg  ?

Thanks !

**Comment by FFY00** [comment]
Can GIMP create SVGs now? I usually use Inkscape, it is pretty mature and should have no problem handling this task.

**Comment by le-grand-mannitout** [comment]
Yes it can but I don't know if I do it wrong or if it's sketchy (lmao ?)..

I am going to use Inkscape, thank you !
